utils/mc/Makefile

```diff
@@ -6,15 +6,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mc
-PKG_VERSION:=4.8.23
-PKG_RELEASE:=2
+PKG_VERSION:=4.8.24
+PKG_RELEASE:=1
 PKG_MAINTAINER:=Dirk Brenken <dev@brenken.org>
 PKG_LICENSE:=GPL-3.0-or-later
 PKG_CPE_ID:=cpe:/a:midnight_commander:midnight_commander
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=http://ftp.midnight-commander.org/
-PKG_HASH:=dd7f7ce74183307b0df25b5c3e60ad3293fd3d3d27d2f37dd7a10efce13dff1c
+PKG_HASH:=859f1cc070450bf6eb4d319ffcb6a5ac29deb0ac0d81559fb2e71242b1176d46
 PKG_BUILD_PARALLEL:=1
 PKG_FIXUP:=autoreconf gettext-version
```

Comment:
I don't think this is needed.

Reply:
Sure? Did you run build tests without these options? In the past that was required and I have currently no time to test such changes. It should be a simple version bump ...